### PR TITLE
feat: ITV-35 add GP section to other support page 

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?
Adapted the other support page so that it includes info about signing up to a gp. This new information is displayed under GP in the review section and is editable . 
The ability to add an additional item has not been coded yet so the add another button does not exist yet. 

image

image

### CODE REVIEW

File: __tests__/features/end-to-end.steps.ts [hunk 1]
```diff
@@ -424,6 +432,7 @@ defineFeature(loadFeature("./end-to-end.feature"), (test) => {
   test("Performing a check", ({ when, then }) => {
     when("I complete a process", async () => {
       const processRef = process.env.TEST_PROCESS_REF;
+      ``;
```
Comment (mrmartineau) [inline __tests__/features/end-to-end.steps.ts:435]: Can this be removed if it is not in use?

Comment (mrmartineau) [inline __tests__/features/end-to-end.steps.ts]: Can this be removed if it is not in use?

File: server/mockData.js [hunk 2]
```diff
@@ -89,7 +89,11 @@ const mockProcessData = {
           photo: { isWilling: "yes", images: [] },
           nextOfKin: { fullName: "njkl" },
           carer: { hasCarer: "no" },
-          otherSupport: { fullName: "mzsl", role: "." },
+          otherSupport: {
+            fullName: "mzsl",
+            role: ".",
+            phoneNumber: "07833109098",
```
Comment (mrmartineau) [inline server/mockData.js]: I know this is only mock data but I think the keys here should match what you have in the schema e.g. have `hasSignedUp` and `notes` - I'm not sure though...

**Review by mrmartineau — CHANGES_REQUESTED** [review]

File: __tests__/features/end-to-end.steps.ts (hunk 1 above)
Comment (LBHABEBIC) [reply __tests__/features/end-to-end.steps.ts]: I'm not sure what this does.

File: server/mockData.js (hunk 2 above)
Comment (LBHABEBIC) [reply server/mockData.js]: yep , I wasn't sure either way.

**Review by LBH-wgreeff — APPROVED** [review]

File: __tests__/features/end-to-end.steps.ts (hunk 1 above)
Comment (mrmartineau) [reply __tests__/features/end-to-end.steps.ts]: it doesn't do anything. let's get rid of it

**Review by mrmartineau — APPROVED** [review]
looks great! thanks for making those changes 👍

**Pull request merged**